build-11.gradle

```diff
@@ -101,6 +99,7 @@ dependencies {
     compile("ch.qos.logback.contrib:logback-jackson:0.1.5")
     compile("net.logstash.logback:logstash-logback-encoder:5.2")
     compile("org.jasypt:jasypt:1.9.3")
+    compile("com.auth0:java-jwt:3.10.3")
```

Comment:
cx-flow uses Java Cryptography Extension (JCE) and not Java JWT (JSON Web Token).
there is no point in adding another package that does the same thing.

Reply:
This library is used to create a JWT ( https://github.com/GregoireW/cx-flow/blob/effaf3b661da0a41fc3cded5131ba837ccc2f6d9/src/main/java/com/checkmarx/flow/service/GitHubAuthService.java#L166 )  it uses JCE in the background.